src/weatherreport/DEVELOPMENT.md

```diff
@@ -0,0 +1,73 @@
+# WeatherReport Development
+
+WeatherReport requires a sane GNU build system and a recent version of
+Erlang.  It has `twig` and `getopt` as dependencies, so those must be
```

Comment:
This is a problem. We don't currently rely on either of these in CouchDB:

```bash
$ git checkout 3.x
Already on '3.x'
Your branch is up to date with 'origin/3.x'.
$ grep -ri twig *
$ find . -name rebar.config\* -exec grep getopt {} \; -print
$
```

That means we need to vendorise these for Apache, meaning bring them in the tree as well (if we're taking over maintaining them) or at least maintain our own mirror repos for them.

getopt is at least BSD licensed, so we can vendorise this in a new Apache CouchDB repo (don't import it into `couchdb`). The README appears to require `rebar3` for building as of 3 years ago. Does it still build with our rebar (2)? It also only lists compatibility thru Erlang 20, I hope that's not accurate, as that'd be a regression in our Erlang support.

Twig is public @ https://github.com/cloudant/twig but has no license stated, does Cloudant/IBM intend to donate this too? Also, do we really need twig instead of the current CouchDB logger? I see no further mention of it in this PR.

Reply:
Yeah, I think most of this file is obsolete. I'll work on fixing up the docs next.

Reply:
This file was obsolete and removed, superseded by the updated README.md